apps/backend/src/app/modules/users/users.service.ts

```diff
@@ -162,10 +163,16 @@ export class UsersService {
       where: { steamID: userData.steamID }
     });
 
-    const input: Prisma.UserUpdateInput | Prisma.UserCreateInput = {
-      alias: userData.alias,
+    const isValidAlias = USER_ALIAS_REGEXP.test(user?.alias);
+
+    const findOrCreateUserData = {
```

Comment:
Why did you put that in a separate variable?

Reply:
Just for read-ability, but I can move it inside the object if you want